pkg/controller/issueapi/logic.go

```diff
@@ -301,6 +301,14 @@ func (c *Controller) issue(ctx context.Context, authApp *database.AuthorizedApp,
 				return err
 			}
 
+			if request.SMSTemplateLabel != "" && currentUser != nil &&
```

Comment:
I feel like "remember my last template" should be entirely in the UI. I don't think we should do this in the core logic. So maybe we move it to the controller method that's _only_ for the UI?